src/options.go

```diff
@@ -2299,6 +2372,12 @@ func ParseOptions() *Options {
 	errorContext = ""
 	parseOptions(opts, os.Args[1:])
 
+	if err := opts.initProfiling(); err != nil {
+		fmt.Fprintln(os.Stderr, "Error: failed to start pprof profiles:", err)
+		util.Exit(1)
```

Comment:
According to the spec, this should be 2.

https://github.com/junegunn/fzf/blob/62963dcefd6d74510ae2692cd0955177bae57543/man/man1/fzf.1#L928-L935

```suggestion
		util.Exit(2)
```

Reply:
Oh, I think we can instead use `errorExit` function.